### DISCUSSION TITLE
Necessary changes to secure our project's future

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Not-Quite-RARBG/main#46] [Announcements]
Hey everyone,

Hope you're all doing great! Let's talk about some crucial updates and clear a few things up, we've noticed some confusion lately, and it's essential to set things straight:

- First, understand that "the team" is not just one person doing everything. The initial website, Reddit post, and README were created by a single user -The owner- before the rest of us joined. We've since noticed this and removed the misleading information from our README. We're a team here and every decision reflects that.
- Second, we've seen the threats of DDoSSing. This kind of behavior is totally unacceptable and has no support from any of us. Rest assured, we've taken action.
- Lastly, concerning the license: this was also handled by the same person previously mentioned. Now, we're revisiting it and will change the license to a GPLv3 one, in line with our commitment to openness and collaboration.

Steps taken to resolve issues and get this project back on track:

- Owner is kicked for his behaviour and access to the GitHub org is revoked
- XXX category is back on the menu, since removing it with his decision and we weren't in support of that

A heads up - since the former founder currently controls our domain and hosting, we're going to be opening up for donations to secure our own resources. If you can, we'd love your support on this. Lastly, we may need to create a new Discord server due to the current ownership. We understand it might be a bit inconvenient, but it's in the best interest of our community.

Thanks for taking the time to read this. As always, we're open for discussions, so feel free to chat or reply below this message.

NQ-RARBG Team

**Comment by joewood9364** [comment]
"will change the license to a GPLv3 one"
So from AGPLV3 to just GPLV3? You are downgrading the license to one which offers less protections for software hosted on network devices (AKA websites)?
If true, this and asking for donations so quickly while using ads are red flags in my opinion.

**Reply by torrentsproxylist** [reply]
Just Close source it already. You guys are tainting piracy, and opensource both at same time.

**Comment by zbynaCool** [comment]
If this is the direction that it's going then please just let the name RARBG die once and for all
